app/src/main/java/fr/free/nrw/commons/bookmarks/pictures/BookmarkPicturesFragment.java

```diff
@@ -59,8 +59,12 @@ public View onCreateView(
             ViewGroup container,
             Bundle savedInstanceState
     ) {
-        View v = inflater.inflate(R.layout.fragment_bookmarks_pictures, container, false);
-        ButterKnife.bind(this, v);
+        binding = FragmentBookmarksPicturesBinding.inflate(inflater, container, false);
+        View v = binding.getRoot();
+        gridView = binding.bookmarkedPicturesList;
```

Comment:
Remove this, this does the work but doesnt give you the optimality. 
This will make you have `import android.widget.GridView;` which is unnecessary as this will make uneccessary import and we will lose the advantage of viewbinding here.

Reply:
Refer PR #5598

Reply:
Well I was using this for tests to work, In your original PR you were not handling tests (checks did not run on your PR), but I have found a better way to handle tests now.